src/lib/y2ntp_client/client/auto.rb

```diff
@@ -0,0 +1,75 @@
+# encoding: utf-8
+
+require "yast"
+require "installation/auto_client"
+
+Yast.import "NtpClient"
+Yast.import "Progress"
+
+require "y2ntp_client/dialog/main"
+
+module Y2NtpClient
+  module Client
+    # This is a client for autoinstallation. It takes its arguments,
+    # goes through the configuration and return the setting.
+    # Does not do any changes to the configuration.
+    class Auto < ::Installation::AutoClient
+      def initialize
+        textdomain "ntp-client"
+      end
+
+      def summary
+        Yast::NtpClient.Summary
+      end
+
+      def import(profile)
+        # as current import only modify existing configuration ( and does not contain complete
```

Comment:
English: "As the current Import only modifies ..."

Hm. If this is true, then the `Import({})` below in `reset` does nothing. Is that correct?

Reply:
no, it do. In fact reset return to previous empty state. So no pools, and default policy and synchronization. Just it does not reread system state. it is questionable if we want reset to system state or to proposed state. But later was done in old days

Reply:
`Import({})` basically sets internal variable to default values, see e.g. https://github.com/yast/yast-ntp-client/pull/101/files#diff-f31befd8011851c773f783bb9afb356fR383 it sets systemd as default.